test/test.getch.ts

```diff
@@ -184,7 +195,7 @@ describe('🥁 configuration options', () => {
       url: `${url}/?james=beckwith&montgomery=scott`,
       params: {james: 'kirk'},
     };
-    const path = '/?james=kirk&montgomery=scott';
+    const path = '/?james=beckwith&montgomery=scott&james=kirk';
```

Comment:
@bcoe this is the part of the change that wigs me out a bit.  Previously, if you defined a querystring parameter both in the `url` and in `params`, we would only ever use one of them when forming the querystring.  This strikes me as being ... incorrect.  But it may be one of those bugs folks are relying on 🤷   My suspicion is that the risk of breaking runtime behavior is relatively low, because who relies on passing querystring parameters both in the url and in a params object.

Reply:
I'd like to do a quick review of axios, request, and fetch, see how they approach this -- I'll do so.

Reply:
I have tested `axios` and `request`, and convinced myself this is the right behavior.